### PR TITLE
Feat earth fallback

### PR DESCRIPTION
**Pull request by mixcas**
- add flat earth fallback for safari and/or iOS
- animate the flat earth

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -366,6 +368,61 @@ Site.Earth = {
...
+
+  },
+
+  canHandleThree: function() {
+    var _this = this;
+
+    // init MobileDetect
+    var md = new MobileDetect(window.navigator.userAgent);
+
+    if (md.is('iOS') || md.is('Safari')) {
```
Comment (mikeyrayvon) [inline src/js/main.js]: this also needs to test for desktop safari

**Comment by mixcas** [comment]
That's testing for any browser on iOS or any safari

File: src/js/main.js (hunk 1 above)
Comment (mikeyrayvon) [reply src/js/main.js]: it isnt working on desktop safari for me

Comment (mixcas) [reply src/js/main.js]: ok, will check

Comment (varffs) [reply src/js/main.js]: Seems like not targeted at desktop anything: https://github.com/hgoebl/mobile-detect.js/issues/3#issuecomment-36220433

Comment (varffs) [reply src/js/main.js]: This plugin isn't great (by its own admission) on desktop. I can't consistently check between Chrome and Safari using it. I was trying to invert the check to test if Chrome of FF so we could just apply the Three to those browsers, but no joy. 

I think a better approach is not to attempt to check browser but see if there are specific pieces of web tech that the Three scene we want uses that are not supported by Safari (on my Safari the scene renders just very very slowly and the audio analyser doesnt seem to return any values) Or perhaps just a desktop and mobile focused detector plugin is a better idea.

**Comment by varffs** [comment]
Errors the onResize handler when the Three scene isn't initiated here: https://github.com/interglobalvision/ondamundial-net/blob/9e4e11b16872ac9812a502b3ad3e541ec810b369/src/js/main.js#L525

Needs to save a state for the Three scene initiated or not and check that conditional

File: src/js/main.js (hunk 1 above)
Comment (mixcas) [reply src/js/main.js]: Can we recheck this?

It successfully tells between desktop chrome and desktop safari, for me.

**Pull request merged**